src/main/resources/static/views/user/UserPage.html

```diff
@@ -0,0 +1,373 @@
+                                           class="form-control input-md input-text">
+                                </div>
+                                <div class="cta-button">
+                                    <button type="submit" class="btn btn-lg btn-block btn-edit">Add Project</button>
+                                </div>
+                            </div>
+                        </fieldset>
+                    </form>
+                </div>
+            </div>
+        </div>
+        <div class="bs-calltoaction bs-calltoaction-success">
+            <div class="row">
+                <div class="col-md-9 cta-contents">
+                    <a href="#!/interests"><h1 class="cta-title">Interests</h1></a>
+                    <div class="cta-desc">
+                        <p><a href="#!/interests">My Interests</a></p>
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                    </div>
+                </div>
+                <div class="col-md-3 form-field">
+                    <form class="form-horizontal" action="" method="" id="addInterest">
+                        <fieldset>
+                            <div class="form-group col-xs-12">
+                                <div>
+                                    <select id="addInterestOption" class="select-input"
+                                            title="What do you want to invest in?">
+                                        <option disabled selected>I want to invest in:</option>
+                                        <option value="1">Startups and Investment Projects</option>
+                                        <option value="2">Venture capital</option>
+                                        <option value="3">Mergers and acquisitions</option>
+                                    </select>
+                                </div>
+                                <div class="cta-button">
+                                    <button type="submit" class="btn btn-lg btn-block btn-edit">Add Interest</button>
+                                </div>
+                            </div>
+                        </fieldset>
+                    </form>
+                </div>
+            </div>
+        </div>
+        <div class="bs-calltoaction bs-calltoaction-success">
+            <div class="row">
+                <div class="col-md-9 cta-contents">
+                    <a href="#!/events"><h1 class="cta-title">Events</h1></a>
+                    <div class="cta-desc">
+                        <p><a href="#!/events">My Events</a></p>
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                    </div>
+                </div>
+                <div class="col-md-3 form-field">
+                    <form class="form-horizontal" action="" method="" id="addEvent">
+                        <fieldset>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a <legend> tag to this fieldset. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)